packages/kit/src/node.js

```diff
@@ -47,3 +49,27 @@ export function getRawBody(req) {
 		});
 	});
 }
+
+/** @type {import('@sveltejs/kit/node').GetRequest} */
+export async function getRequest(base, req) {
+	return new Request(base + req.url, {
+		method: req.method,
+		headers: /** @type {Record<string, string>} */ (req.headers),
+		body: await getRawBody(req)
```

Comment:
doesn't this sort of prevent you from streaming by consuming the whole body?

Reply:
This is where things get a little bit tricky. `node-fetch` doesn't implement `ReadableStream`, it uses node streams for the request body. So in development, preview, and adapter-node apps, you can't actually stream the body by doing this sort of thing:

```js
const reader = request.body.getReader();

let chunk;
while (chunk = await reader.read()) {
  if (chunk.done) {
    return { status: 204 };
  }

  await do_something_with(chunk.value);
}
```

Because of that, there's effectively no difference between exposing the stream and exposing the buffered body — it's going to get buffered regardless, unless people consume the node stream _as_ a node stream, which we'd want to discourage. 

I'm not 100% sure what to do about this — probably something involving proxies and a `ReadableStream` polyfill, until we can drop support for Node <16 — so I figured we could start by getting the basic API in place, then figuring out those details later.

Reply:
maybe just add a TODO for now?
```
TODO: use whatwg web streams when node-fetch v4 is released
// https://github.com/node-fetch/node-fetch/issues/1452
// https://github.com/node-fetch/node-fetch/issues/1454
```